### ISSUE TITLE
[Question] AR processes for multilevel logits and categorical models

### ISSUE THREAD

**Issue by Rain-out** [paul-buerkner/brms#293]
Dear Paul,

Thanks a lot for your work on the brms package. It's been extremely useful so far!

However, I ran into the following error today when specifying an AR(1) process for a multilevel logit:

> Error: ARMA models are not yet implemented for family 'bernoulli'.

Any idea when you will be able to find the time to implement ARMA models for logits and multinomial logits?

**Comment by paul-buerkner** [comment]
I am not sure if this is possible at all for these models, but if you can
point me to an existing implementation or paper I am happy to take a look.

Am 22.11.2017 19:10 schrieb "Rain-out" <notifications@github.com>:
